lib/components/narrative/default/default-itinerary.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import NarrativeItinerary from '../narrative-itinerary'
 import ItinerarySummary from './itinerary-summary'
 import ItineraryDetails from './itinerary-details'
-import TripDetails from '../trip-details'
+import TripDetails from '../connected-trip-details'
```

Comment:
Are we following this convention with the new otp-ui components everywhere? cc @evansiroky. Just want to make sure we're consistent with this.

Reply:
@evansiroky and I have discussed that, I'm following his lead on that.

Reply:
Guess it's settled then.